web/themes/new_weather_theme/templates/layout/page--local.html.twig

```diff
@@ -133,35 +133,36 @@
 
         {{ drupal_block("weathergov_current_conditions") }}
 
-      <tabbed-nav class="display-block position-relative">
-        <div class="tab-buttons padding-bottom-4 padding-top-4 top-0 z-top display-flex flex-row flex-justify position-sticky bg-white top-4 mobile-lg:grid-col-8 desktop:grid-col-6">
+        <tabbed-nav class="display-block position-relative">
+          <div id="tablist-label" class="usa-sr-only">{{ "Weather information sections" | t }}</div>
```

Comment:
@partly-igor Note this text here, which we talked about but perhaps didn't finalize

Reply:
This is good!